### ISSUE TITLE
Servers/Domains List

### ISSUE THREAD

**Issue by RetroJimmyX** [Pugmatt/BedrockConnect#443]
### What happened?

TLDR: The list of domains needs updating and a column indicating the corresponding server name for each domain would be useful on the Github project page.

The Mineplex server appears to have closed for a few months now and is no longer present on the Xbox featyred server list. For somebody adding their own custom DNS entries, it is also not immediately obvious which server (in-game name) corresponds to which domain on the Github project page. I would like, for example, to redirect the Mineville server but I am not sure which domain to use.

Thanks for such a superb project!

### Expected Behaviour?

N/A

### Steps to reproduce.

View main Github project page.

**Comment by Pugmatt** [comment]
Thanks for pointing this out. Mentions of Minplex (and also Pixel Paradise which appears to also have shut down) has been removed from the project files.
